test/src/test/java/org/corfudb/integration/CorfuReplicationReconfigurationIT.java

```diff
@@ -215,6 +227,170 @@ public void testActiveClusterReset() throws Exception {
         assertThat(remainingEntriesToSend).isEqualTo(0L);
     }
 
+    /**
+     * Event table can be cleared after processing the latest force sync event to avoid keeping older events.
```

Comment:
nit: can you please reword the comment to something like 'Verify that event table gets cleared...'.  As it is a test, it will be helpful to know what is being verified.